R/SoilProfileCollection-metadata.R

```diff
@@ -291,3 +291,74 @@ setReplaceMethod("GHL",
                      allowednames = "horizon"
                    )
                  })
+
```

Comment:
Thanks for the new function. I'll write myself a TODO note for adding these examples to the SPC vignette, after discussing to make sure I fully understand usage.

Reply:
See https://github.com/ncss-tech/aqp/pull/268 for context. Not new in this PR